benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -253,6 +271,27 @@ def get_all_reduction_operators():
             reduction_mx_operators[op_name] = mx_operators[op_name]
     return reduction_mx_operators
 
+def get_all_nn_basic_operators():
```

Comment:
Could you be more specific what "nn basic operators"include?

Reply:
1. nn_basic ops as a category was already introduced by Sandeep in his initial opera PR
2. It made sense to keep adding NN layer related ops to this category.
It can be better phrased as nn_layer_ops if that makes sense...